frontend/public/module/k8s/k8s-models.ts

```diff
@@ -72,3 +75,55 @@ export const modelFor = (ref: K8sResourceKindReference) => {
  * NOTE: This will not work for CRDs defined at runtime, use `connectToModels` instead.
  */
 export const allModels = () => k8sModels;
+
+/**
+ * Use this hook to find the model for resources using only group and plural
+ */
+export const useModelFinder = () => {
+  const referenceForGroupVersionPlural = (group: string) => (version: string) => (plural: string) =>
+    [group, version, plural].join('~');
```

Comment:
I'm worried we're going to have a mismatch between `null`, `undefined`, and `''` when there's no group. Can we normalize it to `core`?

```suggestion
    [group || 'core', version, plural].join('~');
```

Reply:
ack. But won't all of them change to `core` since all of them are falsy values?